pawar-2018/pawar2018
# Finishing up NDJ for launch

# What does this pull request do?

Finish up the NDJ for launch.

# How should the reviewer test this pull request?

- Check to see if there's pagination
- Drop down of tags
- Category and tag pages looks like the main index page
- Images looks alright.

# Include the Trello card for this PR, if there is one.

As before.

# Include any screenshots that will help explain this PR.

--

# Any another context you want to provide?

🍗
